engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/MatchIdPredicateContext.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.dsl.predicate;
+
+import java.util.Collection;
+
+/**
+ * The context used when starting to define a match predicate.
```

Comment:
Just noticed this: this is apparently a copy/paste from the match predicate, it should be rephrased or removed.